scripts/Interface/ui/sans_isis/sans_data_processor_gui.py

```diff
@@ -558,6 +575,13 @@ def update_simple_line_edit_field(self, line_edit, value):
             gui_element = getattr(self, line_edit)
             gui_element.setText(str(value))
 
+    def is_multi_period_view(self):
+        return self.multi_period_check_box.isChecked()
+
+    def switch_to_multi_period_view(self):
```

Comment:
Would it be a bit clearer (and potentially more useful) if this method was implemented as a setter? ie
```
def set_multi_period_view_mode(self, mode):
    self.multi_period_check_box.setChecked(mode)
```

Reply:
Changed this as you suggested as I think it is better.